contributed_definitions/yaml/NXmpes_peem.yaml

```diff
@@ -0,0 +1,114 @@
+category: application
+doc: |
+  This is a general application definition for a photo-electron emission or low energy electron microscopy/diffraction experiment.
+type: group
+(NXmpes)NXmpes_peem:
+  (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes_peem]
+    (NXmonitor):
+      exists: recommended
+    (NXinstrument):
+      (NXinsertion_device):
+          exists: optional # only when PEEM uses synchrotron light
+      (NXmonochromator): # incoming_beam_monochromator
+          exists: recommended
+      (NXbeam)beam_probe:
+        exists: optional # as we record it in NXsample
+        \@flux: # that will point to the NXmonitor data field
+        incident_energy(NX_FLOAT): # link to monochromator energy
+        incident_polarization_stokes(NX_NUMBER):
```

Comment:
We already recommend to add `incident_polarization` in `NXmpes`. I would at least make this field recommended, but would also argue to keep either `incident_polarization` **or** `incident_polarization_stokes` and keep this in the parent `NXmpes`.